MetaMask/metamask-extension
# feat(multichain): add BalancesController

## **Description**

This PR adds the Balances Controller that retrieves non EVM balances of accounts. 

## **Related issues**

Fixes https://github.com/MetaMask/accounts-planning/issues/452

## **Manual testing steps**

**This tests requires a btc account for the balances to appear in the state**

1. Start the extension
2. Create a non evm account
3. Wait for interval to pass and see the balances in the state `state.metamask.balances`

## **Screenshots/Recordings**

Not Applicable

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
